cfn-resources/online-archive/docs/partitionfieldview.md

```diff
@@ -36,25 +34,13 @@ _Maximum Length_: <code>700</code>
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
-#### FieldType
-
-Data type of the parameter that that MongoDB Cloud uses to partition data. Partition parameters of type [UUID](http://bsonspec.org/spec.html) must be of binary subtype 4. MongoDB Cloud skips partition parameters of type UUID with subtype 3.
-
-_Required_: No
-
-_Type_: String
-
-_Allowed Values_: <code>date</code> | <code>int</code> | <code>long</code> | <code>objectId</code> | <code>string</code> | <code>uuid</code>
-
-_Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
```

Comment:
this is a breaking change. I thought that this ticket was only about documentation changes

https://jira.mongodb.org/browse/INTMDB-927

Reply:
i know, i realised that te property mention in the docs was not being implemented on the CFN thats why i needed to add it